packages/headless/doc-parser/src/interface-resolver.test.ts

```diff
@@ -147,7 +147,7 @@ describe('#resolveInterfaceMembers', () => {
     resultInterface.addMember(indexer);
     entry.addMember(resultInterface);
 
-    const result = resolveInterfaceMembers(entry, resultInterface);
```

Comment:
My bad, I'm not sure how this made it to `master`...

Reply:
I merged a change to the parser which was not on your branch that added the ability to resolve recursive interfaces. It happens, no worries,